resources/js/store/modules/admin/actions.js

```diff
@@ -133,10 +144,27 @@ export const actions = {
                     not_processed: resp.data.photosNotProcessed,
                     awaiting_verification: resp.data.photosAwaitingVerification
                 });
+
+                context.dispatch('GET_COUNTRIES_WITH_PHOTOS');
             })
             .catch(err => {
                 console.error(err);
             });
-    }
+    },
 
+    /**
+     * Get list of countries that contain photos for verification
+     */
+    async GET_COUNTRIES_WITH_PHOTOS (context)
```

Comment:
I think all admin action names should be prefixed with `ADMIN_` to make it clear this is not a normal user operation